features/step_definitions/employers_steps.rb

```diff
@@ -710,7 +710,7 @@ def employees(*traits)
 Then(/^employer clicks on terminated filter$/) do
   expect(page).to have_content "Select 'Add New Employee' to continue building your roster, or select 'Upload Employee Roster' if you're ready to download or upload the roster template"
   find_by_id('Tab:terminated').click
-  wait_for_ajax
+  sleep(2)
```

Comment:
this is a really bad practice, if the wait_for_ajax function doesn't work, can we search for a selector instead ?

Reply:
i understand; `wait_for_ajax` is nothing but calculated sleep!

> can we search for a selector instead ?

^ it's not failing to find a selector, it's getting `execution expired` error in `wait_for_ajax` and this is only happening in GH actions. it's passing locally. 

let me see, if we need wait_for_ajax in the first place; i'm going to trigger a run without any sleep.

Reply:
you are right, sorry, the function is just a fancy "sleep", I'll take this to a different ticket.